docs/usage.rst

```diff
@@ -136,10 +136,12 @@ All optional attributes are passed as part of the options to ``captureException`
 
     If you're inside a context and your context data includes a ``user`` key, that data will be merged into this.
 
-.. describe:: req
+.. describe:: req/request
 
-    The ``req`` object associated with this event, from a Node http server, Express, Koa, or similar.
-    Will be parsed for request details and user context from ``req.user`` if present.
+    The ``req/request`` object associated with this event, from a Node http server, Express, Koa, or similar.
+    Will be parsed for request details and user context from ``req.user`` if present. It will only pull the data
+    that's handled by the server like method, headers, data, cookies, so you don't need to worry about passing
+    a bloated request object that might contain additional, yet unnecessary data.
```

Comment:
>It will only pull the data that's handled by the server like method, headers, data, cookies, so you don't need to worry about passing a bloated request object that might contain additional, yet unnecessary data.

in my opinion, it's better to enumerate the list here and leave it at that, the second part  (not needing to worry) is implied

Reply:
I'd sooner just name this argument `request` (we accept both, right?), and then mention below that the shorter alias (`req`) is also acceptable. I find labeling this as `req/request` a little confusing.